x-pack/plugins/observability/public/pages/slos/components/slo_list.tsx

```diff
@@ -94,10 +77,10 @@ export function SloList() {
           sloList={sloList}
           loading={isLoadingSloList}
           error={error}
-          onCloned={handleClonedOrDeleted}
-          onCloning={handleCloningOrDeleting}
-          onDeleting={handleCloningOrDeleting}
-          onDeleted={handleClonedOrDeleted}
+          onCloned={noop}
+          onCloning={noop}
+          onDeleting={noop}
+          onDeleted={noop}
```

Comment:
Should we remove these props from the `SloListItems` component if we provide `noop` to them now?